package.json

```diff
@@ -43,10 +45,10 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "serve": "vite preview",
-    "eslint": "eslint --cache src",
-    "eslint-fix": "eslint --fix --cache src",
-    "prettier": "prettier --list-different '**/*.{js,jsx,md}'",
-    "prettier-fix": "prettier --write '**/*.{js,jsx,md}'",
+    "eslint": "eslint --cache \"src/**/*.ts\" \"src/**/*.tsx\" \"src/**/*.js\" \"src/**/*.jsx\"",
+    "eslint-fix": "eslint --fix --cache \"src/**/*.ts\" \"src/**/*.tsx\" \"src/**/*.js\" \"src/**/*.jsx\"",
+    "prettier": "prettier --list-different 'src/**/*.{js,jsx,md,ts,tsx}'",
```

Comment:
adding src here seems like it will bite us in the future if we add another directory that needs to be prettified.